pkg/services/ngalert/api/test-data/am.http

```diff
@@ -10,23 +10,67 @@ GET http://admin:admin@localhost:3000/alertmanager/{{lokiDatasourceID}}/config/a
 GET http://admin:admin@localhost:3000/alertmanager/{{prometheusDatasourceID}}/config/api/v1/alerts
 
 ###
-# grafana requests
+# create AM configuration
+POST http://admin:admin@localhost:3000/alertmanager/{{grafana}}/config/api/v1/alerts
+content-type: application/json
+
+< ./post-user-config.json
+
+###
+# get latest AM configuration
 GET http://admin:admin@localhost:3000/alertmanager/{{grafana}}/config/api/v1/alerts
+content-type: application/json
 
 ###
+# delete AM configuration
 DELETE http://admin:admin@localhost:3000/alertmanager/{{grafana}}/config/api/v1/alerts
 
 ###
-POST http://admin:admin@localhost:3000/alertmanager/{{grafana}}/config/api/v1/alerts
+# create AM alerts
+POST http://admin:admin@localhost:3000/alertmanager/{{grafana}}/api/v2/alerts
 content-type: application/json
 
-< ./post-user-config.json
+###
+# get silences - no silences
+GET http://admin:admin@localhost:3000/alertmanager/{{grafana}}/api/v2/silences?Filter=foo&Filter=bar
 
 ###
+# create silence
 POST http://admin:admin@localhost:3000/alertmanager/{{grafana}}/api/v2/silences
 content-type: application/json
 
-< ./post-silence-data.json
+{
+  "comment": "string",
+  "createdBy": "string",
+  "endsAt": "2021-03-31T14:17:04.419Z",
+  "id": "just-testing",
```

Comment:
You'll need to remove the ID when creating a silence - the `CreateSilence` methods creates and updates.